.babelrc

```diff
@@ -1,9 +1,32 @@
 {
-  "presets": [
-    ["env", {
-      "targets": {
-        "node": "6"
-      }
-    }], "flow"
-  ]
+  "env": {
+    "test": {
+      "presets": [
+        ["env", {
+          "targets": { "node": "current" }
+        }],
+        "flow"
+      ],
+      "plugins": []
+    },
+    "registry": {
```

Comment:
@sergiohgz will provide soon a custom preset that might be shared with plugins to avoid all this logic. https://github.com/verdaccio/core/issues/2